monitor/services/src/main/java/org/datacleaner/monitor/configuration/TenantContextFactory.java

```diff
@@ -29,4 +32,8 @@
     public TenantContext getContext(TenantIdentifier tenant);
     
     public TenantContext getContext(String tenantId);
+
+    public Iterator<RepositoryFolder> getRepositoryFolderIterator();
```

Comment:
This method seems to be only used in unittests which tells me we probably should add it to any interface. And adding it kinda establishes something as a rule which is currently only an implementation detail, namely that each tenant is identified by a repository folder.

Maybe you can just add it to the implementation class as a method, but don't expose it in any interface.